diff --git a/build/index.js b/build/index.js
index 7eacc6661be52e5774c2f597e713eac7616563c9..4fc9c111f42284cf3448b2f9ead21a5e73d5bbb4 100644
--- a/build/index.js
+++ b/build/index.js
@@ -62,7 +62,7 @@ function createRequestHandler(distFolder, { getRoutesManifest: getInternalRoutes
     if (!fs_1.default.existsSync(filePath)) {
         return null;
     }
-    if (/\.[cj]s$/.test(filePath)) {
+    if (/\.c?js$/.test(filePath)) {
         return require(filePath);
     }
     return import(filePath);
diff --git a/build/vendor/vercel.js b/build/vendor/vercel.js
index bf59a827ea24e6a27b258dfe9142c259c9061a7b..466f7e87764f9dbce6340d00ffac2b91956a337e 100644
--- a/build/vendor/vercel.js
+++ b/build/vendor/vercel.js
@@ -59,7 +59,7 @@ function convertRequest(req, res) {
 exports.convertRequest = convertRequest;
 async function respond(res, expoRes) {
     res.statusMessage = expoRes.statusText;
-    res.writeHead(expoRes.status, expoRes.statusText, [...expoRes.headers.entries()]);
+    res.writeHead(expoRes.status, expoRes.statusText, ...expoRes.headers.entries());
     if (expoRes.body) {
         await (0, node_1.writeReadableStreamToWritable)(expoRes.body, res);
     }
