diff --git a/test.d.ts b/test.d.ts
index 25e44ed55af1e874289e233a4ab360431334c01c..2049700fe8ae75f50501272f7a38a45339695f9b 100644
--- a/test.d.ts
+++ b/test.d.ts
@@ -1320,7 +1320,7 @@ declare module "node:test" {
         timers: MockTimers;
     }
     const mock: MockTracker;
-    interface MockFunctionCall<
+    export interface MockFunctionCall<
         F extends Function,
         ReturnType = F extends (...args: any) => infer T ? T
             : F extends abstract new(...args: any) => infer T ? T
