diff --git a/assert.d.ts b/assert.d.ts
index d7e3719186973f8e88e7fece87955e794e9a964c..1dde81704be88dd477efc0863e16bd84d8793ccf 100644
--- a/assert.d.ts
+++ b/assert.d.ts
@@ -506,6 +506,10 @@ declare module "assert" {
          * @since v1.2.0
          */
         function notDeepStrictEqual(actual: unknown, expected: unknown, message?: string | Error): void;
+        /**
+         * @since v23.4.0
+         */
+        function partialDeepStrictEqual<T>(actual: unknown, expected: T, message?: string | Error): void;
         /**
          * Expects the function `fn` to throw an error.
          *
