diff --git a/lib/cli.js b/lib/cli.js
index 5d636413fefae9ab1bdfac927bcbb9f1326ab018..bce3d5fa7dfbfb966ff282b471c00f97d9e750e5 100644
--- a/lib/cli.js
+++ b/lib/cli.js
@@ -13,7 +13,6 @@ const ts = tslib_1.__importStar(require("typescript"));
 const ast = tslib_1.__importStar(require("./ast"));
 const BUILTINS = new Set([
     ...(module_1.Module.builtinModules || []),
-    ...(module_1.Module.builtinModules || []).map(mod => `node:${mod}`),
     `pnpapi`,
 ]);
 function probablyMinified(content) {
@@ -64,6 +63,8 @@ function isValidDependency(ident, { workspace }) {
     return false;
 }
 function checkForUndeclaredDependency(workspace, referenceNode, moduleName, { configuration, report }) {
+    if (moduleName.startsWith("node:"))
+        return;
     if (BUILTINS.has(moduleName))
         return;
     const idents = extractIdents(moduleName);
