diff --git a/build/web/ColorTintFilter.js b/build/web/ColorTintFilter.js
index c62723f8767d4d777379ee063174080619136219..eb186cbd354de2f28e094f211eb4d672dc4be986 100644
--- a/build/web/ColorTintFilter.js
+++ b/build/web/ColorTintFilter.js
@@ -14,9 +14,9 @@ export default function TintColorFilter({ tintColor }) {
     }
     return (<svg style={styles.svg}>
       <defs>
-        <filter id={`expo-image-tint-${tintColor}`} x="0" y="0" width="0" height="0">
-          <feFlood floodColor={tintColor} floodOpacity="1" result="flood"/>
-          <feComposite in="flood" in2="SourceAlpha" operator="in"/>
+        <filter id={`expo-image-tint-${tintColor}`}>
+          <feFlood floodColor={tintColor}/>
+          <feComposite in2="SourceAlpha" operator="atop"/>
         </filter>
       </defs>
     </svg>);
