diff --git a/src/polyfills/require.js b/src/polyfills/require.js
index 5cbacc00f4e04010c9f60cc3af471238e944c3ad..d4c083f7940a0273a31538efa210f7747af37a9c 100644
--- a/src/polyfills/require.js
+++ b/src/polyfills/require.js
@@ -93,7 +93,7 @@ function metroRequire(moduleId) {
 }
 function shouldPrintRequireCycle(modules) {
   const regExps =
-    global[__METRO_GLOBAL_PREFIX__ + "__requireCycleIgnorePatterns"];
+    global[__METRO_GLOBAL_PREFIX__ + "__requireCycleIgnorePatterns"] ?? [/(^|\/|\\)node_modules($|\/|\\)/];
   if (!Array.isArray(regExps)) {
     return true;
   }
