diff --git a/src/polyfills/require.js b/src/polyfills/require.js
index 2dc0c618aae0d3f6d21dd7991ccc180fee879891..754a72600fad34d04bf1bdc3277bc6dbf12d5d69 100644
--- a/src/polyfills/require.js
+++ b/src/polyfills/require.js
@@ -90,7 +90,7 @@ function metroRequire(moduleId) {
 }
 function shouldPrintRequireCycle(modules) {
   const regExps =
-    global[__METRO_GLOBAL_PREFIX__ + "__requireCycleIgnorePatterns"];
+    global[__METRO_GLOBAL_PREFIX__ + "__requireCycleIgnorePatterns"] ?? [/(^|\/|\\)node_modules($|\/|\\)/];
   if (!Array.isArray(regExps)) {
     return true;
   }
