diff --git a/dist/tailwind/web.js b/dist/tailwind/web.js
index 15e2da041b2ac9b6f988a0a10f226d9e7c47ccf5..b34bdf226194757f76071d9c9f6365c76f03ea37 100644
--- a/dist/tailwind/web.js
+++ b/dist/tailwind/web.js
@@ -6,6 +6,7 @@ Object.defineProperty(exports, "__esModule", { value: true });
 const plugin_1 = __importDefault(require("tailwindcss/plugin"));
 const color_1 = require("./color");
 const common_1 = require("./common");
+const safe_area_1 = require("./safe-area");
 const dark_mode_1 = require("./dark-mode");
 const prop_modifier_1 = require("./prop-modifier");
 const switch_1 = require("./switch");
@@ -29,6 +30,7 @@ const preset = {
     },
     plugins: [
         (0, plugin_1.default)(({ addVariant }) => addVariant("web", "&")),
+        safe_area_1.safeArea,
         color_1.color,
         dark_mode_1.darkModeAtRule,
         verify_1.verify,
