diff --git a/out/replicache.d.ts b/out/replicache.d.ts
index 3ce8194a2788b4b36272f1f24fddf2038c1774b2..39b823a4af3abacb2225991d1e954b3f33a16783 100644
--- a/out/replicache.d.ts
+++ b/out/replicache.d.ts
@@ -10,6 +10,7 @@ type JSONValue =
   | string
   | boolean
   | number
+  | object
   | Array<unknown>
   | JSONObject;
 
@@ -35,6 +36,7 @@ type ReadonlyJSONValue =
   | string
   | boolean
   | number
+  | object
   | ReadonlyArray<unknown>
   | ReadonlyJSONObject;
 
